src/CMakeLists.txt

```diff
@@ -250,6 +250,18 @@ if (NOT cereal_FOUND)
   set(OpenMVG_USE_INTERNAL_CEREAL ON)
 endif()
 
+# ==============================================================================
+# minus
+# ==============================================================================
+# - always internal for now (very simple code)
+# ==============================================================================
+set(MINUS_INCLUDE_DIRS 
+  ${CMAKE_CURRENT_SOURCE_DIR}/dependencies/minus/minus/)
+add_subdirectory(dependencies/minus/)
+# Move the following line to multiview/trifocal/CMakeLists.txt
+# But there CMake is ignoring the include directories, a bug?
+include_directories(${MINUS_INCLUDE_DIRS})
```

Comment:
should not be needed since you have `target_include_directories(openMVG_multiview_trifocal`

Reply:
It does work on my computer but on @gandrademachine removing this made it fail, as far as I remember.
In his system the other include in `target_include_directories(openMVG_multiview_trifocal` would simply be ignored (we tested with make VERBOSE=1 and no -I flag for minus would be included). @gandrademachine please test this in your fork by checking out branch trifocal_lib_include, cd to multiview/trifocal, then make and make test.

Reply:
works.

Reply:
@rfabbri , @pmoulon 

I tested as @rfabbri asked and it worked very well. I'm providing the proof that I was running on `trifocal_lib_include` and the results of `make test`:
showing that I was on `trifocal_lib_include` branch: 
```
 $ git branch
  coverity_scan
  develop
  develop_gt_loader
  develop_incremental_v2
  develop_koral
  develop_logger
  develop_matching_lexicographic_ordering
  develop_matching_region_matcher_factory
  develop_spherical_to_cubic
  develop_subposes
  friend_fix
  friend_fix_1618
  gcc-8-support
  gcc-9-support
  gcc-9-travis
  master
  old-develop_trifocal_tmp
  trifocal-app-debug
  trifocal-bug-fix
  trifocal-pr1-v2
  trifocal_lib
* trifocal_lib_include
  trifocal_tmp
  trifocal_tmp2
  trifocal_tmp3
  ts/P2Pt
  ts/develop
  ts/develop_trifocal
  ts/develop_trifocal_tmp
  ts/trifocal-p2pt-insertion
```
showing results after `make test`:
```
$ make test
Running tests...
Test project /home/bielhpp/openMVG-bin/openMVG/multiview/trifocal
    Start 1: openMVG_test_solver_trifocal_three_point
1/3 Test #1: openMVG_test_solver_trifocal_three_point ...   Passed    0.93 sec
    Start 2: openMVG_test_solver_trifocal_metrics
2/3 Test #2: openMVG_test_solver_trifocal_metrics .......   Passed    0.06 sec
    Start 3: openMVG_test_three_view_kernel
3/3 Test #3: openMVG_test_three_view_kernel .............   Passed    7.01 sec

100% tests passed, 0 tests failed out of 3

Total Test time (real) =   8.00 sec

```